sequence/topology/topology.py

```diff
@@ -29,10 +29,10 @@ class Topology(ABC):
         tl (Timeline): the timeline used for simulation
     """
 
-    ALL_CC_CONNECT = "cconnections"
+    ALL_CC_CONNECT = "cconnections"    # NOTE why not name it ALL_C_CONNECT?
```

Comment:
ALL_ClassicalChannel_CONNECT I believe?

Reply:
It seems that here a channel is single-direction, and a connection is bi-direction. So do you think "ALL_ClassicalChannel_CONNECT" is a little strange, because it is saying it is both single-direction and bi-direction?